finix/exceptions.py

```diff
@@ -100,7 +101,16 @@ def __init__(self, status=None, reason=None, http_resp=None):
         if http_resp:
             self.status = http_resp.status
             self.reason = http_resp.reason
-            self.body = http_resp.data
+            self.body = http_resp.data.decode('utf-8')
+            self._body_raw = self.body
+            try:
+                tmp_dict = json.loads(self.body)
+                self.body = ErrorList(tmp_dict)
+            except:
+                if self.body == '':
+                    self.body = ErrorList('')
+                else:
+                    self.body = self.body
```

Comment:
`self.body` for ApiException can be either an ErrorList or a string (if we reach the `else` condition above). May be better to standardize this to always be ErrorList.

Reply:
I want this to always ErrorList so that autocomplete works.

Not sure if autocomplete works if body can be string